docs/reference/runtime/static_memory_optimization.md

```diff
@@ -18,12 +18,12 @@ For a simple program like [Blinky](https://github.com/ARMmbed/mbed-os-example-bl
 | Subtotals           | 53932 |  2312 | 10520 |
 +---------------------+-------+-------+-------+
 ```
-
+<!---Is there a reason this is in backticks?--->
 The `features/frameworks` module includes the Mbed OS test tools, even if you are no longer testing your program. Because of this, you are building one of our test harnesses into every binary. [Removing this module](https://github.com/ARMmbed/mbed-os/pull/2559) saves a significant amount of RAM and flash memory.
 
 #### `Printf` and UART
 
-The linker can also remove other modules that your program does not use. For example, [Blinky's](https://github.com/ARMmbed/mbed-os-example-blinky) `main` program doesn't use `printf` or UART drivers. However, every Mbed OS module handles traces and assertions by redirecting their error messages to `printf` on serial output - forcing the `printf` and UART drivers to be compiled in and requiring a large amount of flash memory.
+The linker can also remove other modules that your program does not use. For example, [Blinky's](https://github.com/ARMmbed/mbed-os-example-blinky) `main` program doesn't use `printf` or UART drivers. However, every Mbed OS module handles traces and assertions by redirecting their error messages to `printf` on serial output. This forces the `printf` and UART drivers to compile in and requires a large amount of flash memory.
```

Comment:
Query: Can we change "to compile in" to "to compile" and keep the meaning?

Reply:
I think so.

Reply:
```suggestion
The linker can also remove other modules that your program does not use. For example, [Blinky's](https://github.com/ARMmbed/mbed-os-example-blinky) `main` program doesn't use `printf` or UART drivers. However, every Mbed OS module handles traces and assertions by redirecting their error messages to `printf` on serial output. This forces the `printf` and UART drivers to compile and requires a large amount of flash memory.
```